src/addBusinessDays/test.ts

```diff
@@ -102,6 +102,147 @@ describe('addBusinessDays', function () {
       // Then we expect to ignore the no-op exception
       assert.deepStrictEqual(result, new Date(2022, 0 /* Jan */, 21))
     })
+
+    it('can override business days with exceptions', function () {
+      // Given business days of Monday - Saturday
+      // And exceptions over one weekend
+
+      // When we add 8 business days
+      const result = addBusinessDays(new Date(2022, 1 /* Feb */, 9), 9, {
+        businessDays: [1, 2, 3, 4, 5, 6],
+        exceptions: {
+          '02/12/22': false,
+          '02/13/22': false,
+          '02/19/22': false,
+        },
+      })
+
+      // Then we expect to have the working Saturdays ignored
+      assert.deepStrictEqual(result, new Date(2022, 1, 22))
```

Comment:
**praise:** thanks for adding the extra tests! Can you add the comment that shows the month in these dates to make them a little easier to read? i.e. `new Date(2022, 1 /* Feb */, 22)`